db/seeds.rb

```diff
@@ -1,5 +1,5 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 
-administrator = FactoryGirl.create(:user, email: 'root@mail.com', password: 'root', password_confirmation: 'root')
+administrator = FactoryGirl.create(:user, email: 'root@mail.com', password: 'password', password_confirmation: 'password')
```

Comment:
https://github.com/zharikovpro/railgun/pull/191/files

when we starting to use standart devise authentificate for password, so the password: root is not valid anymore
https://github.com/zharikovpro/railgun/blob/master/config/initializers/devise.rb#L157